### PR TITLE
Add DossierMigrator that migrates dossier responsibles 

### PR DESCRIPTION
**Pull request by lukasgraf**
Add `DossierMigrator` that migrates `responsible` for dossiers (to be used with `ftw.usermigration`).

@phgross

### CODE REVIEW

File: opengever/usermigration/tests/test_dossier_migrator.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            self.portal, {'old.user': 'new.user'}, 'move')
+        migrator.migrate()
+
+        # Index should be up to date
+        brains = self.catalog(responsible='new.user')
+        self.assertEquals(1, len(brains))
+
+        # Metadata should be up to date
+        brain = self.catalog(path='/'.join(self.dossier.getPhysicalPath()))[0]
+        self.assertEquals('new.user', brain.responsible)
```
Comment (phgross) [inline opengever/usermigration/tests/test_dossier_migrator.py]: Use `from opengever.testing.helpers import obj2brain`

``` python
self.assertEquals('new.user', obj2brain(self.dossier).responsible)
```

File: opengever/usermigration/tests/test_dossier_migrator.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    def test_raises_if_strict_and_user_doesnt_exist(self):
+        migrator = DossierMigrator(
+            self.portal, {'old.user': 'doesnt.exist'}, 'move')
+
+        with self.assertRaises(UserMigrationException):
+            migrator.migrate()
+
+    def test_doesnt_raise_if_not_strict_and_user_doesnt_exist(self):
```
Comment (phgross) [inline opengever/usermigration/tests/test_dossier_migrator.py]: Is there any easy option to check the logging ?

**Comment by phgross** [comment]
Looks good :sparkles:

File: opengever/usermigration/tests/test_dossier_migrator.py (hunk 2 above)
Comment (lukasgraf) [inline opengever/usermigration/tests/test_dossier_migrator.py]: Not sure, simply haven't had the time to test that yet.

**Comment by lukasgraf** [comment]
@phgross updated and rebased.

**Pull request merged**